packages/cli/src/index.ts

```diff
@@ -406,6 +406,7 @@ program
         parseUseDbtListOption,
         true,
     )
+    .option('--raise-errors', 'Raise errors if deploy fails', false)
```

Comment:
We can alsot use the `--ignore-errors` option as `lightdash deploy`. But, it changes the behaviors of the command. That's why I introduced `--raise-errors` to keep the current behavior.